src/main/java/me/coley/recaf/util/UiUtil.java

```diff
@@ -176,9 +182,24 @@ else if(AccessFlag.isAbstract(access))
 			g.getChildren().add(new IconView("icons/modifier/final.png"));
 		if(AccessFlag.isBridge(access) || AccessFlag.isSynthetic(access))
 			g.getChildren().add(new IconView("icons/modifier/synthetic.png"));
+		createAccessToolTips(g, AccessFlag.Type.METHOD, access);
 		return g;
 	}
 
+	private static void createAccessToolTips(Node node, AccessFlag.Type type, int access) {
+		Set<String> accessFlags = AccessFlag.getApplicableFlags(type, access).stream()
+				.map(AccessFlag::getName).collect(Collectors.toSet());
+		Tooltip tooltip = new Tooltip(String.join(", ", accessFlags));
+		node.setOnMouseEntered(event -> {
+			if (!tooltip.getText().isEmpty() && Recaf.getController().config().display().accessFlagsTooltip) {
+				tooltip.show(node, event.getScreenX(), event.getScreenY() + 15);
+			}
+		});
+		node.setOnMouseExited(event -> tooltip.hide());
+		// This has a weird delay
+		// Tooltip.install(node, tooltip);
```

Comment:
I'm pretty sure the idea is that normally tooltips show after hovering the mouse for a short time. It's intentionally designed that way.  Though I do understand why making this show up instantly makes more sense 👍

Reply:
I realized some components already have access flag tooltips, such as in `ClassNodeEditorPane` which makes two pop-ups show up when hovering. I'll fix this in merge.